src/layout/layout.js

```diff
@@ -190,25 +180,35 @@
    *
    * @private
    */
-  MaterialLayout.prototype.screenSizeHandler_ = function() {
-    if (this.screenSizeMediaQuery_.matches) {
-      this.element_.classList.add(this.CssClasses_.IS_SMALL_SCREEN);
 
-      if (this.drawer_) {
-        this.drawer_.setAttribute('aria-hidden', 'true');
-      }
-    } else {
-      this.element_.classList.remove(this.CssClasses_.IS_SMALL_SCREEN);
-      // Collapse drawer (if any) when moving to a large screen size.
-      if (this.drawer_) {
-        this.drawer_.classList.remove(this.CssClasses_.IS_DRAWER_OPEN);
-        this.obfuscator_.classList.remove(this.CssClasses_.IS_DRAWER_OPEN);
 
-        if (this.element_.classList.contains(this.CssClasses_.FIXED_DRAWER)) {
-          this.drawer_.setAttribute('aria-hidden', 'false');
+  function screenSizeHandler(e) {
+    // modified to query dependent elements rather than binding materialLayout to windows media query result
+    var materialLayouts = window.document.querySelectorAll('.mdl-layout')
+    Array.from(materialLayouts).forEach((layout) => {
```

Comment:
It doesn't look like the testrunner support `=>`.

Reply:
It's probably easiest to just use a normal for loop here. I tried updating things to support new language features. If they just used closure compiler it would be fine, but unfortunately they're not using it for most things in the gulpfile.